modules/monitoring-configuring-external-alertmanagers.adoc

```diff
@@ -129,36 +138,35 @@ metadata:
   namespace: openshift-user-workload-monitoring
 data:
   config.yaml: |
-   thanosRuler:
-     additionalAlertmanagerConfigs:
-    - scheme: https
-      pathPrefix: /
-      timeout: "30s"
-      apiVersion: v1
-      bearerToken:
-        name: alertmanager-bearer-token
-        key: token
-      tlsConfig:
-        key: 
-          name: alertmanager-tls
-          key: tls.key
-        cert: 
-          name: alertmanager-tls
-          key: tls.crt
-        ca: 
-          name: alertmanager-tls
-          key: tls.ca
-      staticConfigs:
-      - external-alertmanager1-remote.com
-      - external-alertmanager1-remote2.com
+    thanosRuler:
+      additionalAlertmanagerConfigs:
+      - scheme: https
+        pathPrefix: /
+        timeout: "30s"
+        apiVersion: v1
+        bearerToken:
+          name: alertmanager-bearer-token
+          key: token
+        tlsConfig:
+          key: 
+            name: alertmanager-tls
+            key: tls.key
+          cert: 
+            name: alertmanager-tls
+            key: tls.crt
+          ca: 
+            name: alertmanager-tls
+            key: tls.ca
+        staticConfigs:
+        - external-alertmanager1-remote.com
+        - external-alertmanager1-remote2.com
 ----
+
+. Save the file to apply the changes to the `ConfigMap` object. The new component placement configuration is applied automatically.
+ifndef::openshift-dedicated,openshift-rosa[]
 +
 [NOTE]
 ====
 Configurations applied to the `user-workload-monitoring-config` `ConfigMap` object are not activated unless a cluster administrator has enabled monitoring for user-defined projects.
```

Comment:
```suggestion
Configurations applied to the `user-workload-monitoring-config` `ConfigMap` objects are not activated unless a cluster administrator has enabled monitoring for user-defined projects.
```

Reply:
This is actually just a single ConfigMap object.